conf.d/main

```diff
@@ -19,3 +24,4 @@ rm -rf /var/www/{js,css,images}
 
 # ensure ipv4 forwarding is enabled
 sed -i 's/#* *net.ipv4.ip_forward=.*/net.ipv4.ip_forward=1/g' /etc/sysctl.conf
+
```

Comment:
There is no need for a trailing white space (i.e. a blank line at the end) here. It's not a big deal and I probably wouldn't have even noticed it - except you added it. 

For some reason many of the old scripts (before my time) have this, but there is no good reason for it. So please feel free to remove them from any files you work on.

Reply:
I removed this from the code per PR review comment.